CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # CHANGELOG
-## XX.X.X - 2022-05.XX
+## 20.3.0 - 2022-05.16
 This release adds `us_bank_account` PaymentMethod to PaymentSheet.
```

Comment:
Should we include the actual ticket with `[ADDED]` here?

Reply:
It is below:

* [ADDED][4964](https://github.com/stripe/stripe-android/pull/4964) `us_bank_account` PaymentMethod is now available in PaymentSheet